assets/src/block-templates/homepage/template.js

```diff
@@ -0,0 +1,31 @@
+const {__} = wp.i18n;
+
+const template = () => (
+  [
+    ['core/group', {}, [
+      ['planet4-blocks/carousel-header'],
+      ['planet4-block-templates/issues', {
+        titlePlaceholder: __('The issues we work on', 'planet4-blocks-backend'),
+      }],
+      ['core/spacer', {height: '88px'}],
+      ['planet4-blocks/articles', {
+        article_heading: __('Read our Stories', 'planet4-blocks'),
+      }],
+      ['core/spacer', {height: '56px'}],
+      ['planet4-block-templates/side-image-with-text-and-cta', {
+        title: __('Get to know us', 'planet4-blocks'),
+      }],
+      ['core/spacer', {height: '30px'}],
+      ['planet4-block-templates/side-image-with-text-and-cta', {
+        title: __('We win campaigns', 'planet4-blocks'),
+        mediaPosition: 'right',
+      }],
+      ['core/spacer', {height: '56px'}],
+      ['planet4-blocks/covers'],
+      ['core/spacer', {height: '72px'}],
+      ['gravityforms/form'],
```

Comment:
According to the previous version of the template, this shouldn't be just a Gravity Forms block, but [it should have some text on the side](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1066/files#diff-25f0e4e3db0b77616a6f7b7662e425dc8343525cac2a6b8a478461a2ef08b376L85)
Unless you agreed on this change with the design team?